docs/StempyH5Format.md

```diff
@@ -10,24 +10,27 @@ electron_events
 ├── frames
 └── scan_positions
 stem
-├── bright
-└── dark
+└── images
+frames
```

Comment:
Is this `frames` left over?

Reply:
That's the root `/frames`, which is the raw diffractogram data.

Reply:
ok,make sense